sdk/iOS/test/MSPullSettingsTests.m

```diff
@@ -0,0 +1,63 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import <XCTest/XCTest.h>
+#import "MSPullSettingsInternal.h"
+
+@interface MSPullSettingsTests : XCTestCase
+@end
+
+@implementation MSPullSettingsTests
+
+#pragma mark * Setup and TearDown
+
+- (void)setUp
+{
+    NSLog(@"%@ setUp", self.name);
+}
+
+- (void)tearDown
+{
+    NSLog(@"%@ tearDown", self.name);
+}
+
+#pragma mark * PageSize tests
+
+- (void)testPullSettings_NegativePageSize
+{
+    MSPullSettings *pullSettings = [MSPullSettings new];
+    pullSettings.pageSize = -1;
+
+    XCTAssertEqual(pullSettings.pageSize, MSPullSettings.defaultPageSize, "Incorrect page size");
```

Comment:
Looks like both work (verified that), but from the documentation it is clear it expects NSString. Fixed.